recipe/meta.yaml

```diff
@@ -32,7 +31,7 @@ requirements:
   host:
     - python
     - pip
-    - gfal2
+    - gfal2 2.22.1
```

Comment:
Is pinning the version necessary here ?

Reply:
I messed up, I forgot that `gfal2` is in the global conda-forge pinning so the migration needed to be triggered.

(it doesn't really matter, but this was working around the global pinning)